### DISCUSSION TITLE
Axiom Build command on macOS throwing a 'remove ' or "' exception - but I fixed PATH I think

### DISCUSSION THREAD

**Discussion by PCGEngineer** [axmolengine/axmol#2243] [General]
When I try to create the iOS project using the command in the screenshot it's throwing an exception and not creating anything. My path DID have some quotes in it (thanks to cmake looked like) but I did export PATH="" then export PATH="stuff" and the quotes seem to be gone, see screenshot. I didn't find where the cmake quotes are coming in from, they aren't in my .zshrc but they do reappear whenever I open the terminal. Seems like a silly thing to be fighting....  
<img width="791" alt="Screenshot 2024-11-23 at 10 53 52 PM" src="https://github.com/user-attachments/assets/e5e85286-01dc-4935-b3cf-cc2ba32cc830">

**Comment by halx99** [comment]
enter to pwsh termianl, then `echo $env:PATH`

**Reply by PCGEngineer** [reply]
Thanks! For reference, it wasn't in my .zshrc or in my .zprofile, I had to do
'sudo nano /etc/paths' to find it, cmake had somehow gotten one in with quotes around it
